pycsw/core/pygeofilter_evaluate.py

```diff
@@ -66,7 +66,7 @@ def ilike(self, node, lhs):
         if (str(lhs.prop) == 'dataset.anytext' and
                 self._pycsw_dbtype.startswith('postgres')):
             LOGGER.debug('Kicking into PostgreSQL FTS mode')
-            return text(f"plainto_tsquery('english', '{node.pattern}') @@ anytext_tsvector")  # noqa
+            return text(f"to_tsquery('english', '{node.pattern}') @@ anytext_tsvector")  # noqa
```

Comment:
Can we keep `plainto_tsquery`?  This is what is used in CSW 2/3 support, so it would be valuable for consistency.

Having said this, as we plan to update pycsw to exclusively use pygeoafilter for all queries in a future release, that would be a good opportunity to update PostgreSQL FTS to use `to_tsquery` accordingly.

Reply:
Plainto query does not support fuzzy matching, also consider that #881 is also still an alternative, I closed it because it is less optimal (from multilingual perspective)